config/samples/multicluster_v1alpha1_multiclusterengine.yaml

```diff
@@ -3,4 +3,6 @@ kind: MultiClusterEngine
 metadata:
   name: multiclusterengine-sample
 spec:
-  # Add fields here
\ No newline at end of file
+  # Add fields here
+  overrides:
+    imagePullPolicy: Always
```

Comment:
This file should not be changed, as it causes the `alm-examples` to be updated, which would default these fields if the user creates the MCH in the UI. - https://github.com/stolostron/backplane-operator/pull/91/files#diff-8ae78c7504f293553c9514cd8c4135c797e4fb19bf9cbc5670e97bcf5b85170bR15.

Please remove changes to this file and regenerate the bundle